wiki/Beatmaps/Genre_and_language/en.md

```diff
@@ -0,0 +1,48 @@
+---
+tags:
+  - genres
+  - languages
+  - metadata
+---
+
+# Genre and language
+
+![A screenshot of source, genre, language, and tags on a beatmap page](img/beatmap-info.png "Genre and language are listed on beatmap pages.")
```

Comment:
Could be better to use a screenshot with an actual language like English etc. in it imo, having "Instrumental" on the cover for languages seems a bit off